spec/fixtures/state_file/persona_approved_outputs/2023/ny/javier_return_xmls/xml/submission.xml

```diff
@@ -233,7 +233,7 @@
 </W2StateLocalTaxGrp>
       <StandardOrNonStandardCd>S</StandardOrNonStandardCd>
     </IRSW2>
-          <State1099G xmlns:efile="http://www.irs.gov/efile" documentId="State1099G-12">
+          <State1099G xmlns:efile="http://www.irs.gov/efile" documentId="State1099G-44">
```

Comment:
it is kind of funny that this is basically just dependent on the local db of whoever runs this. not sure i feel motivated to change it though tbh...